changelogs/fragments/6267-homebrew-cask-upgrade-all-greedy.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

I think this is more of a feature though, since the documentation didn't explicit mention that this should work with `upgrade_all` before:
```suggestion
minor_changes:
```